auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthzJwtService.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.auth.jwt;
+
+import static io.stargate.auth.jwt.AuthnJwtService.CLAIMS_FIELD;
+import static io.stargate.auth.jwt.AuthnJwtService.STARGATE_PREFIX;
+
+import io.stargate.auth.AuthorizationService;
+import io.stargate.auth.UnauthorizedException;
+import io.stargate.db.datastore.ResultSet;
+import io.stargate.db.schema.Column;
+import io.stargate.db.schema.Table;
+import java.util.Base64;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.regex.Pattern;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AuthzJwtService implements AuthorizationService {
```

Comment:
Where do we register this service? I could not find the code :)

Reply:
Fixed.